actors/evm/src/interpreter/instructions/call.rs

```diff
@@ -313,3 +312,12 @@ pub fn call_generic<RT: Runtime>(
 
     Ok(U256::from(call_result))
 }
+
+fn effective_gas_limit<RT: Runtime>(system: &System<RT>, gas: U256) -> Option<u64> {
+    let gas = if gas.is_zero() {
+        system.rt.gas_available()
+    } else {
+        std::cmp::min(gas.to_u64_saturating(), system.rt.gas_available())
+    };
+    Some(std::cmp::max((gas * 63) / 64, 1))
```

Comment:
If the user requests _less_ than 63/64 of the gas available, we should respect that.

Reply:
I dont think it will make any difference, the call will fail in that case.
But ok.

Reply:
Oh I see what you mean, basically not reserve anything if he asked for less than 63/64th?
This is getting pedantic.
I personally think we shouldn't reserve anything if the user sets a gas limit, but fine.

Reply:
Implemented your request.

Reply:
https://github.com/filecoin-project/builtin-actors/pull/908/commits/6137ed060994855f16534fa55af3e96c7f495e65

Reply:
So, the issue is that the user might set a limit that's too high for some reason.